types/react-native/index.d.ts

```diff
@@ -8073,6 +8073,26 @@ export interface ImageStoreStatic {
 export type fetch = (url: string, options?: Object) => Promise<any>
 export const fetch: fetch;
 
+interface FormDataHeaders {
+    [name: string]: string
+};
+
+interface FormDataPart1 {
```

Comment:
I guess name like `FormDataPart1` is quite misleading. I'd suggest to use single type `FormData` which is a union type. Exactly like it is done [here](https://github.com/facebook/react-native/blob/master/Libraries/Network/FormData.js#L18).